packages/rpc/src/util.ts

```diff
@@ -2,19 +2,20 @@ import { mbjs, RPC_ENDPOINTS, NEAR_RPC_ENDPOINTS } from '@mintbase-js/sdk';
 import fetch from 'cross-fetch';
 
 
-export type RPC_OPTIONS  = 'lava' | 'near' | 'beta'
-
+export type RPC_OPTIONS  = 'lava' | 'near' | 'beta' | 'fastnear'
 
 export const requestFromNearRpc = async (
   body: Record<string, unknown>,
   network?: string,
   rpc?:  RPC_OPTIONS,
-): Promise<{ result: Record<string, unknown>, error: unknown } | undefined> => {
+  rpcUrl?: string): Promise<{ result: Record<string, unknown>, error: unknown } | undefined> => {
 
   const fetchUrl =  mbjs.keys.nearRpcUrl || RPC_ENDPOINTS[mbjs.keys.rpc][mbjs.keys.network]  || NEAR_RPC_ENDPOINTS[mbjs.keys.network];
```

Comment:
I see 6 different branches to figure out a URL. We determine `fetchUrl` and `rpcAddress`, even if we might not need them. We give three different variable names to what is essentially the same thing. The suffix `Address` is usually used for smart contracts addresses.

This is a good opportunity to make this code less confusing.

Reply:
A way to simplify this is to have ONE way to configure this: Using `mbjs.keys`.